lifetimes/estimation.py

```diff
@@ -96,14 +96,17 @@ def customer_lifetime_value(self, transaction_prediction_model, frequency, recen
         df['frequency'] = frequency
         df['recency'] = recency
         df['T'] = T
+        df['monetary_value'] = monetary_value
```

Comment:
I agree that applying the CLV calculation per individual is super useful. To that end, I would suggest changing `conditional_expected_average_profit` to return individual estimates. Otherwise, there isn't really a reason to have `customer_lifetime_value` inside the Gamma-Gamma model. In fact, it might make a lot of sense to move `customer_lifetime_value` to `utils` to decouple it from a specific model altogether...